docs/manuals/rest/job.md

```diff
@@ -0,0 +1 @@
+!!swagger job.json!!
```

Comment:
I really don't think that these files (manuals/rest/<file>.md) should be commited any time.
They really look like they can be generated at the same time as their json content by tools/generate_rest.py...

Reply:
Sounds good to me.